### DISCUSSION TITLE
Plans on fixing clippy lints?

### DISCUSSION THREAD

**Discussion by eduardorittner** [wezterm/wezterm#5998] [General]
As the title says, are there any plans on fixing them? I was doing a PR for a feature and noticed there are some warnings about feature=cargo-clippy being deprecated, and when I run clippy it doesn't even compile the whole project because it errors on a loop that doesn't loop. I would be willing to try and tackle these clippy lints one by one, just wanted to confirm that these changes would be welcome before starting to work on them. Any thoughts or tips would be greatly appreciated.

**Comment by wez** [comment]
no thanks; I find clippy to be low value in general
